annotations/templates/annotations/project_details.html

```diff
@@ -24,6 +24,9 @@
                     href="{% url "edit_project" project.id %}">
                     <span class="gylphicon glyphicon-pencil"></span> Edit Project
                 </a>
+                <a class="btn btn-sm btn-info" href="{% url 'upload' %}">
+                <span class="gylphicon glyphicon-pencil"></span> Upload
```

Comment:
Is this for importing appellations? If so, it should say 'Import Appellations'.

Reply:
changed